src/ZPublisher/Converters.py

```diff
@@ -23,20 +23,15 @@
 
 
 def field2string(v):
-    """Converts value to native strings.
-
-    So always to `str` no matter which Python version you are on.
-    """
-    if hasattr(v, 'read'):
-        return v.read()
-    elif isinstance(v, bytes):
```

Comment:
I am not sure if there should be a deprecation (maybe we could add it to Zope 4) or at least a warning (here in Zope 5).

Reply:
Well, that is up to you do decide.

I certainly agree it is a good idea to add a deprecation warning for Zope 4.

At the current state of this PR, a warning in Zope 5 is no longer reasonable, as the code path is gone.

If you insist on a deprecation warning in Zope 5, I'd need to create a new PR, and stall this one until Zope 6? I am not sure if this is necessary.

Reply:
I am fine with the current solution.